pkg/scheduler/framework/plugins/noderesources/resource_allocation.go

```diff
@@ -35,6 +35,7 @@ type resourceAllocationScorer struct {
 	useRequested bool
 	scorer       func(requested, allocable []int64) int64
 	resources    []config.ResourceSpec
+	podRequest   map[v1.ResourceName]int64
```

Comment:
`podRequest` differs in each scheduling cycle and it seems weird to be stored in `resourceAllocationScorer` and reflash it in every scheduling cycle. (because I expect the fields contain the global configuration which is not related to the specific scheduling cycle.)

Reply:
So, it's gonna look better to pass `podRequest` as the parameter of `score` func, I think.